action.yml

```diff
@@ -32,6 +32,10 @@ inputs:
     description: 'path to a .bandit file that supplies command line arguments'
     required: false
     default: 'DEFAULT'
+  config_path:
+    description: 'path to a YAML or TOML file that supplies command line arguments'
```

Comment:
:thinking: it might be worth mentioning the [official documentation](https://bandit.readthedocs.io/en/latest/config.html)

Reply:
Also to be tracked by #8